examples/repeat_entry.rb

```diff
@@ -11,7 +11,7 @@
 
 puts "Ok, you did it."
 
-pass = ask("<%= @key %>:  ") do |q|
+pass = ask("<%= key %>:  ") do |q|
```

Comment:
No sure if this need to be `@key` or `key` any of this are declared on this file, and README don't have a example with a ERB like.